third_party/ngen/ngen_core.hpp

```diff
@@ -300,6 +300,153 @@ static inline constexpr14 PlatformType getPlatformType(ProductFamily family) {
     return PlatformType::Unknown;
 }
 
+static inline constexpr14 PlatformType getPlatformType(uint32_t deviceID) {
+    // A list of device ids can be found in:
+    // https://github.com/intel/compute-runtime/blob/master/shared/source/dll/devices/devices_base.inl
+    // only NAMEDDEVICE ids are included.
+    switch(deviceID) {
```

Comment:
Looks like PTL is missing:
https://github.com/intel/compute-runtime/blob/d7ca6568fc70582bd88fbda239efdef57ec03763/shared/source/dll/devices/devices_base.inl#L10

I think we need `DEVICE` ids as well.